dynamodb/src/test/scala/zio/dynamodb/ExecutorSpec.scala

```diff
@@ -18,7 +30,10 @@ object ExecutorSpec extends DefaultRunnableSpec with DynamoDBFixtures {
   )
 
   override def spec: ZSpec[Environment, Failure] =
-    suite("Batching")(crudSuite, scanAndQuerySuite, batchingSuite).provideLayer(DynamoDBExecutor.test)
+    suite("Executor spec")(
+      suite("Batching")(crudSuite, scanAndQuerySuite, batchingSuite).provideLayer(DynamoDBExecutor.test),
+      batchRetries
```

Comment:
ah, I did not spot this before. The existing `ExecutorSpec` is testing code in the DSL layer and providing a fake for the executor. The new `batchRetries` is testing code in the live executor `DynamoDBExecutorImpl` so should probably live in its own spec file as they are testing different modules. Probably the original `ExecutorSpec` should be re-named to something else like `BatchingDslSpec` (I am rubbish when it comes to naming stuff)

Reply:
Okay, I can move the original tests to a new file and keep these new tests in here